docs/source/examples/de_sscan.ipynb

```diff
@@ -0,0 +1,1505 @@
+       " 'diode': array([1., 1., 1., 1., 1., 0., 1., 2., 2., 3., 3., 3., 3., 1., 1., 1., 1.,\n",
+       "        1., 1., 1., 1.], dtype=float32)}"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def get_sscan_data():\n",
+    "    npts = scan1.current_point.get()\n",
+    "    data = {\n",
+    "        \"m1\": scan1.positioners.p1.array.get()[:npts],\n",
+    "        \"I0\": scan1.detectors.d01.array.get()[:npts],\n",
+    "        \"diode\": scan1.detectors.d02.array.get()[:npts],\n",
+    "    }\n",
+    "    return data\n",
+    "\n",
+    "get_sscan_data()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Setup and run same scan from bluesky\n",
+    "\n",
+    "Repeat the scan from the previous examples, but use bluesky to configure\n",
+    "`scan1`.  It will be useful to connect the motor, the scaler, and two the scaler\n",
+    "channels."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ophyd import EpicsMotor\n",
+    "from ophyd.scaler import ScalerCH\n",
+    "\n",
+    "m1 = EpicsMotor(f\"{IOC}m1\", name=\"m1\")\n",
+    "scaler1 = ScalerCH(f\"{IOC}scaler1\", name=\"scaler1\")\n",
+    "m1.wait_for_connection()\n",
+    "scaler1.wait_for_connection()\n",
+    "\n",
+    "# for convenience\n",
+    "I0 = scaler1.channels.chan02.s\n",
+    "diode = scaler1.channels.chan04.s"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can supply the count time per point and scan range parameters as arguments to\n",
+    "the setup.  After setting the counting time for the scaler, the next step in the\n",
+    "setup is to clear any existing configuration of `scan1` using its `reset()`\n",
+    "method.\n",
+    "\n",
+    "In a bluesky plan, we'll need to use `apstools.plans.run_blocking_function` with\n",
+    "that `reset()` method.\n",
+    "\n",
+    "Finally, we'll setup `scan1` with the EPICS PV names to be used."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def setup_scan1(start, finish, npts, ct=1):\n",
+    "    yield from bps.mv(scaler1.preset_time, ct)  # counting time/point\n",
+    "    yield from run_blocking_function(scan1.reset)\n",
+    "    yield from bps.sleep(0.2)  # arbitrary wait for EPICS record processing\n",
+    "\n",
+    "    # positioners\n",
+    "    yield from bps.mv(\n",
+    "        scan1.number_points, npts,\n",
+    "        scan1.positioners.p1.start, start,\n",
+    "        scan1.positioners.p1.end, finish,\n",
+    "        scan1.positioners.p1.readback_pv, m1.user_readback.pvname,\n",
+    "        scan1.positioners.p1.setpoint_pv, m1.user_setpoint.pvname,\n",
+    "    )\n",
+    "    # triggers (push scaler count button at each point)\n",
```

Comment:
Does it need to be a different `yield from` or is it just for readability? Would it make a difference if it was a single `yield from`?